components/GraphDiagram/GraphDiagramDownloadButton.tsx

```diff
@@ -45,7 +45,7 @@ function downloadPng() {
     return;
   }
 
-  const canvas = $.create<HTMLCanvasElement>('canvas');
+  const canvas = document.createElement('canvas');
```

Comment:
Another helper was `$.create`, which especially considering the required type assertion, was actually counter-productive

Reply:
Agreed. npmgraph was my first real foray into Typescript.  APIs like `$.create()` weren't particularly nuanced as a result. 🤷